solutions/LC matrix-block-sum.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: lc-MatrixBlockSum
+source: LeetCode
+title: Matrix Block Sum
+author: Jesse Choe
+---
+
+**Time Complexity:** $\mathcal O(NM)$.
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+```cpp
+class Solution {
+public:
+    vector<vector<int>> matrixBlockSum(vector<vector<int>>& mat, int K) {
+        int n = mat.size(), m = mat[0].size();
+        int arr[n][m];
```

Comment:
variable length arrays are not c++ standard

Reply:
uhhhhhhh i just made it this way b/c i'm not comfortable using vector<vector<int>> lol.

Reply:
make the sizes of the array the max possible from the bounds of the problem